client/webpack.config.node.js

```diff
@@ -163,6 +163,7 @@ const webpackConfig = {
 			BUILD_TIMESTAMP: JSON.stringify( new Date().toISOString() ),
 			COMMIT_SHA: JSON.stringify( commitSha ),
 			'process.env.NODE_ENV': JSON.stringify( bundleEnv ),
+			__i18n_text_domain__: JSON.stringify( 'default' ),
```

Comment:
Do we need the `JSON.stringify` here?

Reply:
This was taken from the [client webpack.config](https://github.com/Automattic/wp-calypso/blob/trunk/client/webpack.config.js#L295) and is similar to the [config for editing-toolkit](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/webpack.config.js#L86). 
It needs to use `'"default"'` to replace `__i18n_text_domain__`.